io.catenax.fleet.vehicles/2.1.0/Vehicles.ttl

```diff
@@ -0,0 +1,403 @@
+
+:OemShortNameTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic samm-c:Text ;
+   samm-c:constraint :WorldManufacturerInformationCodeLength .
+
+:EnginesList a samm-c:List ;
+   samm:preferredName "Engines"@en ;
+   samm:description "A list of all installed engines in the vehicle"@en ;
+   samm:dataType :Engine .
+
+:EquipmentsList a samm-c:List ;
+   samm:preferredName "Equipment List"@en ;
+   samm:description "List of equipments installed in the vehicle."@en ;
+   samm:dataType :Equipment .
+
+:CountryCodeCharacteristic a samm:Characteristic ;
+   samm:preferredName "Country Code Characteristic"@en ;
+   samm:description "ISO 3166-1 alpha-3 ? three-letter country codes "@en ;
+   samm:dataType xsd:string .
+
+:CountryCodeRegularExpression a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "Country Code Regular Expression"@en ;
+   samm:description "Regular Expression that ensures a three-letter code "@en ;
+   samm:see <https://www.iso.org/iso-3166-country-codes.html> ;
+   samm:value "^[A-Z][A-Z][A-Z]$" .
+
+:WorldManufacturerInformationCodeLength a samm-c:LengthConstraint ;
+   samm:preferredName "Length restriction for WMI"@en ;
+   samm:description "Restricts the length of wmiCode to exactly 3 chars."@en ;
+   samm:see <https://vpic.nhtsa.dot.gov/> ;
+   samm-c:minValue "3"^^xsd:nonNegativeInteger ;
+   samm-c:maxValue "3"^^xsd:nonNegativeInteger .
+
+:Engine a samm:Entity ;
+   samm:preferredName "Engine Entity"@en ;
+   samm:description "Describing one installed engine"@en ;
+   samm:properties ( :engineId [ samm:property :engineDescription; samm:optional true ] [ samm:property :engineSeries; samm:optional true ] [ samm:property :serialNumber; samm:optional true ] [ samm:property :size; samm:optional true ] [ samm:property :power; samm:optional true ] [ samm:property :engineProductionDate; samm:optional true ] [ samm:property :installDate; samm:optional true ] [ samm:property :nhtsaFuelType; samm:optional true ] ) .
+
+:Equipment a samm:Entity ;
+   samm:preferredName "Equipment"@en ;
+   samm:description "One equipment of the vehicle."@en ;
+   samm:properties ( :equipmentIdentifier [ samm:property :equipmentDescription; samm:optional true ] [ samm:property :group; samm:optional true ] ) .
+
+:engineId a samm:Property ;
+   samm:preferredName "Engine ID:"@en ;
+   samm:description "OEM-specific identifier/type of the installed engine"@en ;
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "CKBY" .
+
+:engineDescription a samm:Property ;
+   samm:preferredName "Engine Description"@en ;
+   samm:description "Description of the engine"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2.0 TDI" .
+
+:engineSeries a samm:Property ;
+   samm:preferredName "Vehicle engine series"@en ;
+   samm:description "Engine series as defined by OEM"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "EA189" .
+
+:serialNumber a samm:Property ;
```

Comment:
`partInstanceId` 
@jacewski-bosch

Reply:
As this model is used in a not Industry Core related use case, we decided to leave this as is for now. In a future version this should be resolved, which will, be guardrailed by a yet to define MS2 requirement. (#642 )
@agg3fe Please proceed with MS3 in the next meeting.